waterdata/templates/monitoring_location.html

```diff
@@ -6,6 +6,11 @@
         {{ json_ld }}
     </script>
     {% endif %}
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ=="crossorigin=""/>
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js" integrity="sha512-/Nsx9X4HebavoBvEBuyp3I7od5tA0UzAxs+j83KgC8PU0kgB4XiK4Lfe4y4cgBtaRJQEIFCW+oC506aPT2L1zw==" crossorigin=""></script>
+    <script src="https://unpkg.com/esri-leaflet@2.1.2/dist/esri-leaflet.js"
+    integrity="sha512-ouokQ1RIIoqPTZKwlapdxHO5VWFoAi8wE+SwhSX89Ifac0w3p+H2da4oqHvRsBTIpNLewzAZU7gRVDFXyXcfjA=="
+    crossorigin=""></script>
```

Comment:
This is definitely debatable - rather than pulling in via npm, this is using a Cloudflare hosted CDN for Leaflet 1.3.1. If we ditch this approach, perhaps we should create a separate app bundle for the Leaflet JS, so we can only load it on the pages where it's needed.

An option, also, would be for the component's initialization code to use something similar to jQuery's $.getScript() function, so the markup doesn't need to track these dependencies.

Reply:
I think soon we should probably think about having different bundles for different pages. Is this usually done? Or I don't see the difference between Leaflet and D3 in terms of there are pages in the application that don't use D3 either. I think we should be consistent, so I would prefer that these be brought in with npm.

Reply:
Yes we could do that. We'd want to have a common bundle for D3, etc, so they don't get downloaded multiple times. Possibly also with some lazy loading.